challenges-webapp/src/main/webapp/vue-app/challenges/components/Challenges.vue

```diff
@@ -20,7 +20,10 @@
     </v-toolbar>
     <template v-if="displayChallenges">
       <div class="pl-2 pt-5">
-        <challenges-list :challenges="challenges" @edit-challenge="editChallenge($event)" />
+        <challenges-list
+          :challenges="challenges"
+          @edit-challenge="editChallenge($event)"
+          @create-announce="createAnnounce($event)" />
```

Comment:
```suggestion
          @create-announce="createAnnounce($event)" />
```
We must delete `createAnnounce` and make the drawer handling its events inside it with `this.$root.$on('open-announcement-drawer'` and `document.addEventListener('open-announcement-drawer'`